src/lints/trait_method_added.ron

```diff
@@ -0,0 +1,61 @@
+SemverQuery(
+    id: "trait_method_added",
+    human_readable_name: "pub trait method added",
+    description: "A non-sealed public trait added a new method without a default implementation",
+    required_update: Major,
+    lint_level: Deny,
+    reference_link: Some("https://doc.rust-lang.org/cargo/reference/semver.html#trait-new-item-no-default"),
+    query: r#"
+    {
+        CrateDiff {
+            current {
+                item {
+                    ... on Trait {
+                        visibility_limit @filter(op: "=", value: ["$public"]) @output
+                        sealed @filter(op: "!=", value: ["$true"])
```

Comment:
This one's tricky, and could have gone either way. So far, we've been treating "the trait got sealed" and "new items requiring implementation got added" as separate issues which should both lint. That's how the other lints in this category work too.

For consistency, we should remove this.
```suggestion
```

Would you mind adding a test case that covers this as well?